src/components/navbar/account-link.cjsx

```diff
@@ -9,5 +9,5 @@ module.exports = React.createClass
   render: ->
     return null unless CurrentUserStore.getProfileUrl()
     <li>
-      <a href={CurrentUserStore.getProfileUrl()} target='_blank'>My Account</a>
+      <a href={CurrentUserStore.getProfileUrl()} target='_blank' onClick={if @props.onClick then @props.onClick}>My Account</a>
     </li>
```

Comment:
In React, setting a prop to an undefined value is the same as not setting it at all. Accordingly this could just be `onClick={@props.onClick}`